### ISSUE TITLE
"Wallet Updated At" timestamp is in the future

### ISSUE THREAD

**Issue by okkar** [guydavis/machinaris#171]
**Describe the bug**
"Wallet Updated At" values are in the future. In the example, the server time zone is set to "Asia/Singapore".

Server time: 13:31:04 (UTC+8)
Wallet Updated At: 21:30

**To Reproduce**

1. Go to "Wallet" page.
2. Compare "Wallet Updated At" values with the current time at the bottom of the page.

**Expected behavior**

Wallet Updated At <= current time

**System setup:**
 - OS: Ubuntu 20.04
 - Docker version number: 20.10.7
 - Machinaris branch: main
 - Machinaris version number: 0.5.0

**Config**
<details><summary>full configuration</summary>

```
# paste your Docker run or Docker compose here
{
    "AppArmorProfile": "docker-default",
    "Args": [
        "./entrypoint.sh"
    ],
    "Config": {
        "AttachStderr": false,
        "AttachStdin": false,
        "AttachStdout": false,
        "Cmd": null,
        "Domainname": "",
        "Entrypoint": [
            "bash",
            "./entrypoint.sh"
        ],
        "Env": [
            "API_SETTINGS_FILE=/root/.chia/machinaris/config/api.cfg",
            "AUTO_PLOT=false",
            "controller_api_port=8927",
            "controller_host=localhost",
            "controller_web_port=8926",
            "farmer_address=null",
            "farmer_pk=",
            "farmer_port=8447",
            "FLASK_APP=/machinaris/main.py",
            "FLASK_ENV=production",
            "keys=/root/.chia/mnemonic.txt",
            "mode=fullnode",
            "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/chia-blockchain/venv/bin:/flax-blockchain/venv/bin",
            "plots_dir=/plots/plots-dst:/plots/ds1815-chia-plots-volume1:/plots/ds1815-chia-plots-volume2:/plots/ds1815-chia-plots-volume3:/plots/farm/hdd01:/plots/farm/hdd02:/plots/farm/hdd03:/plots/farm/hdd04:/plots/farm/ds410-chia-plots-volume1",
            "pool_pk=",
            "testnet=false",
            "TZ=Asia/Singapore",
            "WEB_SETTINGS_FILE=/root/.chia/machinaris/config/web.cfg",
            "XDG_CONFIG_HOME=/root/.chia",
            "blockchains=chia,flax",
            "pool_contract_address=",
            "flax_farmer_port=6885"
        ],
        "ExposedPorts": {
            "6885/tcp": {},
            "6888/tcp": {},
            "8444/tcp": {},
            "8447/tcp": {},
            "8555/tcp": {},
            "8926/tcp": {},
            "8927/tcp": {}
        },
        "Hostname": "ubuntu",
        "Image": "ghcr.io/guydavis/machinaris",
        "Labels": {},
        "OnBuild": null,
        "OpenStdin": false,
        "StdinOnce": false,
        "Tty": true,
        "User": "",
        "Volumes": {
            "/id_rsa": {},
            "/plots/ds1815-chia-plots-volume1": {},
            "/plots/ds1815-chia-plots-volume2": {},
            "/plots/ds1815-chia-plots-volume3": {},
            "/plots/farm": {},
            "/plots/plots-dst": {},
            "/plotting": {},
            "/root/.chia": {}
        },
        "WorkingDir": "/chia-blockchain"
    },
    "Created": "2021-07-11T15:29:12.434142248Z",
    "Driver": "overlay2",
    "ExecIDs": [
        "68566b46dea3ea2cf4ecdd8b5a2fa57d3d257c04c937ec859f078e261ea7e545",
        "cf99f1a659bc014824ba9283d84cb7898dd61a273064d1741e347d1d6cd4ca6b",
        "2b59e8b70f8ea8001bf09814933dd3e219a53751bab518d8587194a0212850ce",
        "1f15be6c89b1ccc3398fd9899a48caa074229713e6a7de9a50db3bd6e4f3f5b1",
        "318119ac7778280eedfc7ecde49d5733f204ee26d5d67a27c2181d6872e33d79",
        "6f6db04a453905ce98c2cb160b00660fffe489b88b83ae77ad7d2cdaa39f0e9e",
        "b67becee0d0d55ae0655f196ff93d311c5cdcd6dfc187a5d72b56216e1de271b",
        "80c58d05ecf6941224ca3723fad2aebf00d6b2a8242036b3d238dd4f6deff80c"
    ],
    "GraphDriver": {
        "Data": {
            "LowerDir": "/var/lib/docker/overlay2/037665570bd2f4e91c7a7329440218cb5588ec00255126f8dbc08c2878d043a6-init/diff:/var/lib/docker/overlay2/ac153b4e5b62556d95c8f5f442eda013c33256f1b033e14d14dca4df8f8080f2/diff:/var/lib/docker/overlay2/799f34802999626e6897bc4cc8de7ef01851e7463783b894d60a0d4912cee8dc/diff:/var/lib/docker/overlay2/bdb60e86fb85fb6cf3989d91ccfe51855e7abce15fa41109929be5c30915b297/diff:/var/lib/docker/overlay2/bd77b553a507824b8a022b463b8f0e03b6389e338ea39ee9d304e613d6e82050/diff:/var/lib/docker/overlay2/2656b78d6eee7c42ed466df3cb578756696e57060b10f611d76e9d00229c1bcf/diff",
            "MergedDir": "/var/lib/docker/overlay2/037665570bd2f4e91c7a7329440218cb5588ec00255126f8dbc08c2878d043a6/merged",
            "UpperDir": "/var/lib/docker/overlay2/037665570bd2f4e91c7a7329440218cb5588ec00255126f8dbc08c2878d043a6/diff",
            "WorkDir": "/var/lib/docker/overlay2/037665570bd2f4e91c7a7329440218cb5588ec00255126f8dbc08c2878d043a6/work"
        },
        "Name": "overlay2"
    },
    "HostConfig": {
        "AutoRemove": false,
        "Binds": [
            "/home/okkar/plots-dst:/plots/plots-dst",
            "/mnt/ssd:/plotting",
            "4111d542f0b7f33d3e0580758f4bddaa1185c11eb98328e3142cc448ec641b8f:/id_rsa",
            "/home/okkar/.machinaris:/root/.chia",
            "/mnt/ds1815-chia-plots-volume1:/plots/ds1815-chia-plots-volume1",
            "/mnt/farm:/plots/farm",
            "/mnt/ds1815-chia-plots-volume2:/plots/ds1815-chia-plots-volume2",
            "/mnt/ds1815-chia-plots-volume3:/plots/ds1815-chia-plots-volume3"
        ],
        "BlkioDeviceReadBps": null,
        "BlkioDeviceReadIOps": null,
        "BlkioDeviceWriteBps": null,
        "BlkioDeviceWriteIOps": null,
        "BlkioWeight": 0,
        "BlkioWeightDevice": [],
        "CapAdd": [
            "AUDIT_WRITE",
            "CHOWN",
            "DAC_OVERRIDE",
            "FOWNER",
            "FSETID",
            "KILL",
            "MKNOD",
            "NET_BIND_SERVICE",
            "NET_RAW",
            "SETFCAP",
            "SETGID",
            "SETPCAP",
            "SETUID",
            "SYS_CHROOT"
        ],
        "CapDrop": [
            "AUDIT_CONTROL",
            "BLOCK_SUSPEND",
            "DAC_READ_SEARCH",
            "IPC_LOCK",
            "IPC_OWNER",
            "LEASE",
            "LINUX_IMMUTABLE",
            "MAC_ADMIN",
            "MAC_OVERRIDE",
            "NET_ADMIN",
            "NET_BROADCAST",
            "SYSLOG",
            "SYS_ADMIN",
            "SYS_BOOT",
            "SYS_MODULE",
            "SYS_NICE",
            "SYS_PACCT",
            "SYS_PTRACE",
            "SYS_RAWIO",
            "SYS_RESOURCE",
            "SYS_TIME",
            "SYS_TTY_CONFIG",
            "WAKE_ALARM"
        ],
        "Cgroup": "",
        "CgroupParent": "",
        "CgroupnsMode": "host",
        "ConsoleSize": [
            0,
            0
        ],
        "ContainerIDFile": "",
        "CpuCount": 0,
        "CpuPercent": 0,
        "CpuPeriod": 0,
        "CpuQuota": 0,
        "CpuRealtimePeriod": 0,
        "CpuRealtimeRuntime": 0,
        "CpuShares": 0,
        "CpusetCpus": "",
        "CpusetMems": "",
        "DeviceCgroupRules": null,
        "DeviceRequests": null,
        "Devices": [],
        "Dns": [],
        "DnsOptions": [],
        "DnsSearch": [],
        "ExtraHosts": [],
        "GroupAdd": null,
        "IOMaximumBandwidth": 0,
        "IOMaximumIOps": 0,
        "IpcMode": "private",
        "Isolation": "",
        "KernelMemory": 0,
        "KernelMemoryTCP": 0,
        "Links": null,
        "LogConfig": {
            "Config": {},
            "Type": "json-file"
        },
        "MaskedPaths": [
            "/proc/asound",
            "/proc/acpi",
            "/proc/kcore",
            "/proc/keys",
            "/proc/latency_stats",
            "/proc/timer_list",
            "/proc/timer_stats",
            "/proc/sched_debug",
            "/proc/scsi",
            "/sys/firmware"
        ],
        "Memory": 0,
        "MemoryReservation": 0,
        "MemorySwap": 0,
        "MemorySwappiness": null,
        "NanoCpus": 0,
        "NetworkMode": "bridge",
        "OomKillDisable": false,
        "OomScoreAdj": 0,
        "PidMode": "",
        "PidsLimit": null,
        "PortBindings": {
            "6885/tcp": [
                {
                    "HostIp": "",
                    "HostPort": "6885"
                }
            ],
            "6888/tcp": [
                {
                    "HostIp": "",
                    "HostPort": "6888"
                }
            ],
            "8444/tcp": [
                {
                    "HostIp": "",
                    "HostPort": "8444"
                }
            ],
            "8447/tcp": [
                {
                    "HostIp": "",
                    "HostPort": "8447"
                }
            ],
            "8926/tcp": [
                {
                    "HostIp": "",
                    "HostPort": "8926"
                }
            ],
            "8927/tcp": [
                {
                    "HostIp": "",
                    "HostPort": "8927"
                }
            ]
        },
        "Privileged": false,
        "PublishAllPorts": false,
        "ReadonlyPaths": [
            "/proc/bus",
            "/proc/fs",
            "/proc/irq",
            "/proc/sys",
            "/proc/sysrq-trigger"
        ],
        "ReadonlyRootfs": false,
        "RestartPolicy": {
            "MaximumRetryCount": 0,
            "Name": "no"
        },
        "Runtime": "runc",
        "SecurityOpt": null,
        "ShmSize": 67108864,
        "UTSMode": "",
        "Ulimits": null,
        "UsernsMode": "",
        "VolumeDriver": "",
        "VolumesFrom": null
    },
    "HostnamePath": "/var/lib/docker/containers/ca491efafa0f1683949b1043215b6eda066676e4f8482a68328452dc991ae0de/hostname",
    "HostsPath": "/var/lib/docker/containers/ca491efafa0f1683949b1043215b6eda066676e4f8482a68328452dc991ae0de/hosts",
    "Id": "ca491efafa0f1683949b1043215b6eda066676e4f8482a68328452dc991ae0de",
    "Image": "sha256:1c6cfe82a80bafd6160c7be7770741e6d7f74824af8ee49854310cbff759bcf5",
    "LogPath": "/var/lib/docker/containers/ca491efafa0f1683949b1043215b6eda066676e4f8482a68328452dc991ae0de/ca491efafa0f1683949b1043215b6eda066676e4f8482a68328452dc991ae0de-json.log",
    "MountLabel": "",
    "Mounts": [
        {
            "Destination": "/plots/ds1815-chia-plots-volume1",
            "Mode": "",
            "Propagation": "rprivate",
            "RW": true,
            "Source": "/mnt/ds1815-chia-plots-volume1",
            "Type": "bind"
        },
        {
            "Destination": "/plots/farm",
            "Mode": "",
            "Propagation": "rprivate",
            "RW": true,
            "Source": "/mnt/farm",
            "Type": "bind"
        },
        {
            "Destination": "/plots/ds1815-chia-plots-volume2",
            "Mode": "",
            "Propagation": "rprivate",
            "RW": true,
            "Source": "/mnt/ds1815-chia-plots-volume2",
            "Type": "bind"
        },
        {
            "Destination": "/plots/ds1815-chia-plots-volume3",
            "Mode": "",
            "Propagation": "rprivate",
            "RW": true,
            "Source": "/mnt/ds1815-chia-plots-volume3",
            "Type": "bind"
        },
        {
            "Destination": "/plots/plots-dst",
            "Mode": "",
            "Propagation": "rprivate",
            "RW": true,
            "Source": "/home/okkar/plots-dst",
            "Type": "bind"
        },
        {
            "Destination": "/plotting",
            "Mode": "",
            "Propagation": "rprivate",
            "RW": true,
            "Source": "/mnt/ssd",
            "Type": "bind"
        },
        {
            "Destination": "/id_rsa",
            "Driver": "local",
            "Mode": "z",
            "Name": "4111d542f0b7f33d3e0580758f4bddaa1185c11eb98328e3142cc448ec641b8f",
            "Propagation": "",
            "RW": true,
            "Source": "/var/lib/docker/volumes/4111d542f0b7f33d3e0580758f4bddaa1185c11eb98328e3142cc448ec641b8f/_data",
            "Type": "volume"
        },
        {
            "Destination": "/root/.chia",
            "Mode": "",
            "Propagation": "rprivate",
            "RW": true,
            "Source": "/home/okkar/.machinaris",
            "Type": "bind"
        }
    ],
    "Name": "/machinaris",
    "NetworkSettings": {
        "Bridge": "",
        "EndpointID": "5a3373ee1ace6599881deac7de7a34c5c6bdb0212c4477af45edf48a595a5796",
        "Gateway": "172.17.0.1",
        "GlobalIPv6Address": "",
        "GlobalIPv6PrefixLen": 0,
        "HairpinMode": false,
        "IPAddress": "172.17.0.2",
        "IPPrefixLen": 16,
        "IPv6Gateway": "",
        "LinkLocalIPv6Address": "",
        "LinkLocalIPv6PrefixLen": 0,
        "MacAddress": "02:42:ac:11:00:02",
        "Networks": {
            "bridge": {
                "Aliases": null,
                "DriverOpts": null,
                "EndpointID": "5a3373ee1ace6599881deac7de7a34c5c6bdb0212c4477af45edf48a595a5796",
                "Gateway": "172.17.0.1",
                "GlobalIPv6Address": "",
                "GlobalIPv6PrefixLen": 0,
                "IPAMConfig": {},
                "IPAddress": "172.17.0.2",
                "IPPrefixLen": 16,
                "IPv6Gateway": "",
                "Links": null,
                "MacAddress": "02:42:ac:11:00:02",
                "NetworkID": "8cce9bd5b8e8786128378f3528d3e43a9fbda80b95e573b8484815c448780305"
            }
        },
        "Ports": {
            "6885/tcp": [
                {
                    "HostIp": "0.0.0.0",
                    "HostPort": "6885"
                },
                {
                    "HostIp": "::",
                    "HostPort": "6885"
                }
            ],
            "6888/tcp": [
                {
                    "HostIp": "0.0.0.0",
                    "HostPort": "6888"
                },
                {
                    "HostIp": "::",
                    "HostPort": "6888"
                }
            ],
            "8444/tcp": [
                {
                    "HostIp": "0.0.0.0",
                    "HostPort": "8444"
                },
                {
                    "HostIp": "::",
                    "HostPort": "8444"
                }
            ],
            "8447/tcp": [
                {
                    "HostIp": "0.0.0.0",
                    "HostPort": "8447"
                },
                {
                    "HostIp": "::",
                    "HostPort": "8447"
                }
            ],
            "8555/tcp": null,
            "8926/tcp": [
                {
                    "HostIp": "0.0.0.0",
                    "HostPort": "8926"
                },
                {
                    "HostIp": "::",
                    "HostPort": "8926"
                }
            ],
            "8927/tcp": [
                {
                    "HostIp": "0.0.0.0",
                    "HostPort": "8927"
                },
                {
                    "HostIp": "::",
                    "HostPort": "8927"
                }
            ]
        },
        "SandboxID": "41608fc0700ba85f33b164d17e5fbe451852c25b4479efac273ddfc104fb1cc8",
        "SandboxKey": "/var/run/docker/netns/41608fc0700b",
        "SecondaryIPAddresses": null,
        "SecondaryIPv6Addresses": null
    },
    "Path": "bash",
    "Platform": "linux",
    "Portainer": {
        "ResourceControl": {
            "Id": 7,
            "ResourceId": "ca491efafa0f1683949b1043215b6eda066676e4f8482a68328452dc991ae0de",
            "SubResourceIds": [],
            "Type": 1,
            "UserAccesses": [],
            "TeamAccesses": [],
            "Public": false,
            "AdministratorsOnly": true,
            "System": false
        }
    },
    "ProcessLabel": "",
    "ResolvConfPath": "/var/lib/docker/containers/ca491efafa0f1683949b1043215b6eda066676e4f8482a68328452dc991ae0de/resolv.conf",
    "RestartCount": 0,
    "State": {
        "Dead": false,
        "Error": "",
        "ExitCode": 0,
        "FinishedAt": "0001-01-01T00:00:00Z",
        "OOMKilled": false,
        "Paused": false,
        "Pid": 90869,
        "Restarting": false,
        "Running": true,
        "StartedAt": "2021-07-11T15:29:12.714652898Z",
        "Status": "running"
    }
}
```

</details>

**Additional context & screenshots**
![image](https://user-images.githubusercontent.com/3801075/125238068-436f7e80-e319-11eb-814c-f271181618df.png)

**Comment by guydavis** [comment]
Please paste your Docker run or Docker compose here.  Thanks!

**Comment by okkar** [comment]
I've updated the details in the original post.

**Comment by guydavis** [comment]
Now available in [v0.5.1 release](https://github.com/guydavis/machinaris/releases/tag/0.5.1).
